### ISSUE TITLE
Set up firewall on CentOS server

### ISSUE THREAD

**Issue by arikalfus** [arikalfus/nextcloud-ansible#10]

**Comment by arikalfus** [comment]
Using the `iptables` module in Ansible
https://docs.ansible.com/ansible/latest/iptables_module.html

**Comment by arikalfus** [comment]
Actually there is a `firewalld` module we can use
https://docs.ansible.com/ansible/latest/firewalld_module.html

**Comment by arikalfus** [comment]
Looks like we want:
```ansible
- firewalld:
    service: http
    permanent: true
    state: enabled

- firewalld:
    service: https
    permanent: true
    state: enabled

- service: name=firewalld state=restarted
```

**Comment by arikalfus** [comment]
@abefleming do we have to enable HTTP?

**Comment by arikalfus** [comment]
Implemented with #16
